docs/scenarios/library-authoring/fastening-libraries-to-rc2.md

```diff
@@ -0,0 +1,70 @@
+# How to manage package dependency versions for RC2
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers what you need to know about package versions for authoring libraries.
+
+## Glossary
+
+**Metapackage** - A NuGet package that represents a set of NuGet packages.  They're organized in a way that makes sense for general library and app development.
+
+**Trimming** - The act of removing the packages you do not depend on from a metapackage.  The end result is that your library only depends on packages it uses.  See guidance in [How to trim your package dependencies](trimming.md).
+
+## Fasten your dependencies to RC2
+
+When creating libraries with .NET Core RC2, it's important that you fasten your dependencies to RC2.  This means two things.  First, you should only depend on packages marked as final for .NET Core RC2.  Second, each package should come from the same build. This means every package should have this pattern: `<package_version>-rc2-<build_number>`.
```

Comment:
Suggestion:
Second, each package should come from the same build; that is, every package should have this pattern...